app/models/guidance.rb

```diff
@@ -13,12 +13,10 @@ class Guidance < ActiveRecord::Base
   ##
   # Associations
   belongs_to :guidance_group
-#  belongs_to :question
+  belongs_to :question
```

Comment:
yes. I thought I had it commented out but may have added it back in prior to removing those dropdowns from the guidance pages. I'll remove it.